<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QListLoadClass</class>
 <widget class="QDialog" name="QListLoadClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>272</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>272</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Load File</string>
  </property>
  <widget class="QLabel" name="pic1Label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>272</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>480</width>
     <height>272</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>480</width>
     <height>272</height>
    </size>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="pcsp.qrc">:/../../Resources/Resources/pic1.png</pixmap>
   </property>
  </widget>
  <widget class="QListWidget" name="icon0List">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>160</width>
     <height>272</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>272</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>160</width>
     <height>272</height>
    </size>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="64">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="64">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>240</red>
         <green>240</green>
         <blue>240</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="autoScrollMargin">
    <number>82</number>
   </property>
   <property name="showDropIndicator" stdset="0">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>144</width>
     <height>80</height>
    </size>
   </property>
   <property name="textElideMode">
    <enum>Qt::ElideNone</enum>
   </property>
   <property name="verticalScrollMode">
    <enum>QAbstractItemView::ScrollPerPixel</enum>
   </property>
   <property name="horizontalScrollMode">
    <enum>QAbstractItemView::ScrollPerPixel</enum>
   </property>
   <property name="layoutMode">
    <enum>QListView::Batched</enum>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="viewMode">
    <enum>QListView::IconMode</enum>
   </property>
   <property name="uniformItemSizes">
    <bool>true</bool>
   </property>
   <property name="batchSize">
    <number>6</number>
   </property>
  </widget>
  <widget class="QLabel" name="descriptionLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>211</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="selectBut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBut">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="pcsp.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>icon0List</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>QListLoadClass</receiver>
   <slot>displayPic1(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>160</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>icon0List</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>QListLoadClass</receiver>
   <slot>displayDesc(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectBut</sender>
   <signal>pressed()</signal>
   <receiver>QListLoadClass</receiver>
   <slot>loadButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>360</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>336</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelBut</sender>
   <signal>pressed()</signal>
   <receiver>QListLoadClass</receiver>
   <slot>cancelButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>451</x>
     <y>257</y>
    </hint>
    <hint type="destinationlabel">
     <x>453</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>displayPic1(QListWidgetItem*)</slot>
  <slot>displayDesc(QListWidgetItem*)</slot>
  <slot>loadButton()</slot>
  <slot>cancelButton()</slot>
 </slots>
</ui>
